src/ecrc-frontend/src/components/page/informationreview/InformationReview.test.js

```diff
@@ -230,6 +262,25 @@ describe("InformationReview Component", () => {
     expect(history.location.pathname).toEqual("/criminalrecordcheck/error");
   });
 
+  test("Sets error and redirects to error page when checkApplicantForPrevCRC fails with something other than 400", async () => {
```

Comment:
i think we have a test case where it mocks a 400 return right? we should change the text on that test to say
```
Sets error and does not redirect to error page when checkApplicantForPrevCRC fails with 400
```

Reply:
so it makes sense and is consistent with this test you added/the actual functionality

Reply:
All the tests that do not check share are currently running in the 400 state, only the ones that directly test share are in 200, and this one is in 500 to make sure it redirects when we get a non 400 error.

Reply:
so can we add a share test case which is 400?